CONTRIBUTING.md

```diff
@@ -21,9 +29,29 @@ your change directly to the repository:
 
 Running clang-format is required, except in the [test folder](https://github.com/oneapi-src/oneDPL/tree/main/test).
 
-# License
+# Validation Testing
 
-oneDPL is licensed under the terms in [LICENSE](https://github.com/oneapi-src/oneDPL/blob/release_oneDPL/licensing/LICENSE.txt).
-By contributing to the project, you agree to the license and copyright terms therein and
-release your contribution under these terms.
+oneDPL validation tests can be configured through CMake to run on different devices including CPU and GPU by specifying the oneDPL backend to use and
+the device type to be used if the DPC++ backend is being tested. The oneTBB and OpenMP backends for oneDPL can be used when the device type is CPU or HOST.
+
+To run oneDPL validation tests with the oneDPL build system you will need CMake and CTest installed. The following example shows how to build and run tests
+using the gpu on your local system using the Intel® oneAPI DPC++ Compiler.
+
+1. Configure the Makefiles by running the following command from the [test folder](https://github.com/oneapi-src/oneDPL/tree/main/test).
+```
+cmake -DCMAKE_CXX_COMPILER=icpx -DCMAKE_CXX_STANDARD=17 -DONEDPL_BACKEND=dpcpp -DONEDPL_DEVICE_TYPE=gpu -DCMAKE_BUILD_TYPE=release ..
+```
+
+2. Build the tests
+```
+cmake --build . --target build-onedpl-tests # specify a specific test name (e.g., reduce.pass) to build a single test
```

Comment:
If the above comment is applied, then we should put `--build build_gpu_tests`. For the CTest command below we can say to navigate to the build directory.

Reply:
Done